packages/components/src/components/hive-navigation/index.tsx

```diff
@@ -71,23 +69,46 @@ export function HiveNavigation({
   const themeConfig = useThemeConfig() as ReturnType<typeof useThemeConfig> | undefined;
   const Search = themeConfig?.search?.component;
 
+  const mobileMenu = useMenu();
+  console.log({ mobileMenu });
+
   const isHive = productName === 'Hive';
 
+  const logo = (
+    <Anchor href="/" className="-m-2 flex items-center rounded-md p-2">
+      <HiveCombinationMark className="text-green-1000 dark:text-neutral-200" />
+    </Anchor>
+  );
```

Comment:
you could just move this outside as a separate component given it doesn't change

Reply:
good idea